recipes/extra-cmake-modules/all/test_package/foo.h

```diff
@@ -0,0 +1,5 @@
+int success() 
```

Comment:
is this supposed to be merged?

Reply:
yes, there is no C++ interface to test with, dummy example is okay